usr/share/rear/restore/NBKDC/default/40_restore_backup.sh

```diff
@@ -0,0 +1,16 @@
+LogPrint "
+
+The System is now ready for restore. Please start the restore task from the NovaBACKUP DataCenter Central Management!
+
+! Remember that the restore target must be set to '/mnt/local' !
+
+For further documentation see the following link:
+
+ http://www.novastor.com/help-html/dc/en-US/index.html
+
+
+after the restore has finished quit this wizard with the command 'exit' to continue.
+"
+
+rear_shell "Did you restore the backup to /mnt/local ? Are you ready to continue recovery ?"
```

Comment:
Well, in https://github.com/rear/rear/blob/master/usr/share/rear/restore/REQUESTRESTORE/default/20_prompt_user_to_start_restore.sh we use the same thing and BACKUP=REQUESTRESTORE is still the default backup setting.

I wonder if that actually still works. Maybe it doesn't and we never noticed because nobody uses that?